crates/subspace-networking/src/behavior.rs

```diff
@@ -1,5 +1,7 @@
 pub(crate) mod custom_record_store;
 pub(crate) mod persistent_parameters;
+#[cfg(test)]
+mod tests;
```

Comment:
Modules above imports, all imports together, please

Reply:
The new module seems to be above imports 👀 .

Reply:
My bad :facepalm: